pkg/resource/parameter_group/update_pre_build.go

```diff
@@ -0,0 +1,78 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package parameter_group
+
+import (
+	"context"
+	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
+	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
```

Comment:
Please split and group the standard library, internal and external imports. https://github.com/golang/go/wiki/CodeReviewComments#imports

```go
import (
        "context"

	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
	svcsdk "github.com/aws/aws-sdk-go/service/memorydb"
)
```